src/wechaty/user/message.py

```diff
@@ -440,9 +440,17 @@ async def forward(self, to: Union[Room, Contact]):
     def date(self) -> datetime:
         """
         Message sent date
+        Python2.7: https://docs.python.org/2.7/library/datetime.html#datetime.datetime
+        Python3+ ：https://docs.python.org/3.7/library/datetime.html#datetime.datetime
+        for datetime.fromtimestamp. It’s common for this to be restricted to years from 1970 through 2038. 
+        2145888000 is 2038-01-01 00:00:00 UTC for second
+        2145888000 is 1970-01-26 04:04:48 UTC for millisecond
         :return:
         """
-        time = datetime.fromtimestamp(self.payload.timestamp)
+        if self.payload.timestamp > 2145888000:
```

Comment:
It's great to see our Python code can also support the timestamp in both seconds and milliseconds here!

In order to make sure our code is correct, I'd like to request that we should add related unit tests for this new feature so that we can make sure it works as expected!

Reply:
I've tested this by my bot and it worked well for a time(2days). But I've no idea about how to make it unit tests as your wish. Let me know what kind of unit tests you wanta, want to add unit tests python code or something else?

Reply:
here's a intersting things: when I use `self.payload.time_stamp`, it will hit an issue about No attribute 'time_stamp', but `self.payload.timestamp` will go though. I found @wj-Mcat is using `self.payload.time_stamp`. Can someboy know this?

Reply:
I think this unit test is for time-data process logic which is not related with wechaty. So , you can refactor this time-data process logic data to help/utils function,  and make small unit test code for it. @huan  how do you think about ?